File: code_wars/kyu_6/005-Detect_Pangram.js
// ###### Solution 1
function isPangram(string) {
  const letters = Array.from({ length: 26 }, (_, index) =>
    String.fromCharCode(`${index + 97}`)
  );

  string = string.toLowerCase();

  for (let i = 0; i < letters.length; i++) {
    if (!string.includes(letters[i])) return false;
  }

  return true;
}

// ###### Video Link:
// ###### Problem Link: https://www.codewars.com/kata/545cedaa9943f7fe7b000048/train/python

File: code_wars/kyu_6/005-Detect_Pangram.py
###### Solution 1
def is_pangram(st):
    letters = [chr(i) for i in range(ord("a"), ord("z") + 1)]
    st = {*st.lower()}

    for letter in letters:
        if letter not in st:
            return False
    return True


###### Solution 2
def is_pangram(st):
    letters = {chr(i) for i in range(ord("a"), ord("z") + 1)}
    st = {*st.lower()}

    return letters.issubset(st)


###### Video Link:
###### Problem Link: https://www.codewars.com/kata/545cedaa9943f7fe7b000048/train/python
